chia/util/misc.py

```diff
@@ -18,19 +18,20 @@ class VersionedBlob(Streamable):
     blob: bytes
 
 
-def format_bytes(bytes: int) -> str:
-    if not isinstance(bytes, int) or bytes < 0:
+def format_bytes(bytes_value: int, effective: bool = False) -> str:
```

Comment:
Seems strange to add this `effective` parameter where the only difference is that it adds an `e` to the label? Why not just add the `e` in the places where you use it with `effective=True` because it seems in those places you anyway also add `(effective)` to the string.